.github/workflows/release.yml

```diff
@@ -30,28 +34,28 @@ jobs:
           git config user.email "actions@github.com"
           git config user.name "GitHub Actions"
  
- #     - name: Setup Git user
- #       uses: fregante/setup-git-user@v2 
- #       
-#      - name: Setup SSH Keys and known_hosts
-#        env:
-#          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
-#        run: |
-#            mkdir -p ~/.ssh
-#            ssh-keyscan github.com >> ~/.ssh/known_hosts
-#            ssh-agent -a $SSH_AUTH_SOCK > /dev/null
-#            ssh-add - <<< "${{ secrets.SSH_PRIVATE_KEY }}"
+      - name: Set Release Version
+        id: vars
+        shell: bash
+        run: |
+          echo "VERSION=${{ github.event.inputs.version }}" >> $GITHUB_OUTPUT
+          mvn versions:set -DnewVersion=${{ github.event.inputs.version }}-SNAPSHOT
+          git commit -m "Releasing version ${{ github.event.inputs.version }}" pom.xml
+          git push origin main
```

Comment:
Probably should not be called explicitly, Maven Release Plugin will do the push anyway